### ISSUE TITLE
Keep a common namespace to avoid conflicts with applications

### ISSUE THREAD

**Issue by tchoutri** [bitwalker/exirc#75]
In the project I'm using ExIrc in, one of the applications is called `IRC`. 
I noticed that the `ExIrc` namespace isn't systematically used, and that `Irc.Commands` and `IrcMessage` are defined in `ExIrc`. This is highly confusing because ExIrc uses 3 different namespaces, and the risks of conflicts are super high.

**Comment by shymega** [comment]
@tchoutri I agree completely with you on this. It would make a _lot more_ sense to keep a common namespace, so perhaps a PR could be made to merge this in.. provided @bitwalker doesn't have any objections?

**Comment by bitwalker** [comment]
I have no objections - exirc is from waaaaay back in early Elixir days, before there was even community consensus on how namespacing was to be handled (and was something I pushed for after some clashes between Timex and some other date/time libs at the time). I thought exirc had already been namespaced properly, but I guess I never actually checked. I'm open to a PR to address this any time!

**Comment by tchoutri** [comment]
I'll take care of that once #74 is merged into master :)
